views/js/front.js

```diff
@@ -18,6 +18,8 @@
  */
 
 $(document).ready(function() {
+    formatGdprForm();
+    
```

Comment:
```suggestion
$(function() {
    formatGdprForm();
    
```
Using `$(document).ready` is deprecated since jQuery 3.0. `$(function() { })` or `jQuery(function() { })` should be used.